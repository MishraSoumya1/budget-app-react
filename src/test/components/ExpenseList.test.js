import React from "react";
import { ExpenseList } from "../../components/ExpenseList";
import { shallow } from "enzyme";
import { expenses } from "../fixtures/expenseData";

describe("expenselist is rendering the component properly", () => {
  test("validate list rendered properly", () => {
    const wrapper = shallow(<ExpenseList expenses={expenses} />);
    expect(wrapper).toMatchSnapshot();
  });
});
