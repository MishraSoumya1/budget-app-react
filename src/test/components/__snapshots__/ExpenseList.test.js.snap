// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expenselist is rendering the component properly validate list rendered properly 1`] = `
<div>
  <h1>
    Expense List
  </h1>
  <Connect(ExpenseListItem)
    amount={195}
    createdAt={0}
    description="Gum"
    id="1"
    key="1"
    note=""
  />
  <Connect(ExpenseListItem)
    amount={1095}
    createdAt={-1000}
    description="Rent"
    id="2"
    key="2"
    note=""
  />
  <Connect(ExpenseListItem)
    amount={450}
    createdAt={1000}
    description="Loan"
    id="3"
    key="3"
    note=""
  />
</div>
`;
